### PR TITLE
Updated README including adding badge, and deleted unused file

### PR DESCRIPTION
**Pull request by okeefm**
Pull requests into the Cypress Validation Utility require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: N/A
- [x] Internal ticket links to this PR N/A
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases N/A
- [x] Tests have been run locally and pass N/A

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
 [![GitHub version](https://badge.fury.io/gh/projectcypress%2Fcypress-validation-utility.svg)](https://badge.fury.io/gh/projectcypress%2Fcypress-validation-utility)
+[![Build Status](https://travis-ci.com/projectcypress/cypress-validation-utility.svg?branch=master)](https://travis-ci.com/projectcypress/cypress-validation-utility)
 
 # Cypress Validation Utility
 
-Upload any QRDA Category 1 or Category 3 document. This application will validate it against the 2014 QRDA Errata release and optionally against the CMS Combined IG.
+Upload any QRDA Category 1 or Category 3 document. This application will validate it against various QRDA IGs, based on the reporting program and document type chosen.
```
Comment (mayerm94) [inline README.md]: Should we list the QRDA versions supported instead of just saying "various"?
